tutorials/01_introductory/sample_plots.py

```diff
@@ -133,17 +134,17 @@
 Bar charts are simple to create using the :func:`~matplotlib.pyplot.bar`
```

Comment:
Same about removing simple. `Use :func:`~matplotlib.pyplot.bar` command to make bar charts, which including customization:` (though won't hold you/hold up the pr to this since you didn't write it in the first place)